client/my-sites/upgrades/domain-management/style.scss

```diff
@@ -850,20 +854,18 @@ ul.email-forwarding__list {
 	}
 }
 
-.name-servers {
-	.icann-verification {
-		.button {
-			margin-bottom: 15px;
+.icann-verification {
```

Comment:
I think this CSS should live with the component. It might be time to create a new directory for the component and the CSS.